src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,514 @@
+sessions acquired in other subjects.
+Similarly, in situations where different data types are obtained over several
+visits (for example, FDG PET on one day followed by amyloid PET a couple days after)
+those can be grouped in one session.
+Please see the difference with the `run-<label>` below.
+
+With respect to the `task-<label>`, data is arranged in a similar way as task-based
+and resting state BOLD fMRI data.
+In case of studies using combined PET/fMRI, subject-specific tasks may be carried out
+during the acquisition within the same session.
+Therefore, it is possible to specify `task-<label>` in accordance with the fMRI data.
+For more information please see https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#task-including-resting-state-imaging-data.
+
+In case of studies with multiple acquisitions per subject using different tracers,
+the `acq-<label>` must be used to distinguish between different tracers.
+Please keep in mind that the label used is arbitrary and each file requires
+a separate JSON sidecar with details of the tracer used (see below).
+Examples are `acq-18FFDG` for fludeoxyglucose, `acq-11CPIB` for Pittsburgh compound B, etc.
+
+The reconstruction key (`rec-<label>`) has four reserved values:
+`acdyn`, for reconstructions with attenuation correction of dynamic data;
+`acstat`, for reconstructions with attenuation correction of static data;
+`nacdyn`, for reconstructions without attenuation correction of dynamic data;
+and `nacstat`, for reconstructions without attenuation correction of static data.
+Further details regarding reconstruction are in the _pet.json file.
+In case of multiple reconstructions of the data with the same type,
+we allow for using a number after the `<label>` in order to distinguish,
+for example `rec-acdyn1` and `rec-acdyn2`.
+
+The run entity is used if one scan type/contrast is repeated multiple times
+within the same scan session/visit.
+If several scans of the same modality are acquired they MUST be indexed
+with a key-value pair: `_run-1`, `_run-2`, `_run-3` etc. (only integers are allowed as run labels).
+When there is only one scan of a given type the run key MAY be omitted.
+An example of this would be two consecutive scans performed within the same session,
+for example two short FDG scans right after each other.
+It is our assumption that the `run-<label>` is used in PET for the cases
+where the subject does not leave the scanner.
+Otherwise, we refer to the `ses-<label>` definition.
+
+In addition to the imaging data a `_pet.json` sidecar file needs to be provided.
+The included metadata are divided into sections described below.
+
+### PET sidecar JSON (`*_pet.json`)
+
+#### Information
+
+This section is mandatory and contains general information about the imaging experiment.
+Some of the fields are marked optional (MAY), for example anaesthesia; for those fields
```

Comment:
```suggestion
Some of the fields are marked optional (OPTIONAL), for example anaesthesia; for those fields
```

I think in the other specs this is called OPTIONAL?
@sappelhoff @effigies could you confirm?

Reply:
OPTIONAL and MAY are used interchangeably, but for this case, I would do the following:

```suggestion
Some of the fields are marked OPTIONAL, for example anaesthesia; for those fields
```

Reply:
Thanks @sappelhoff! I've seen MAY written inside text, but in the tables always OPTIONAL

Reply:
Has this change been integrated? Or it will only show at the end? (sorry I'm not very familiar with the github reviewing process)